Cypress/cypress/pages/getStarted.js

```diff
@@ -3,5 +3,9 @@
 Cypress.Commands.add("getStarted", (newPassword, projectName) => {
   cy.get("[data-cy=inputPassword] input").clear().type(newPassword);
   cy.get("[data-cy=confirmInputPassword] input").clear().type(newPassword);
+  cy.intercept({
+        url: 'http://localhost:3000/update/details',
```

Comment:
same for endpoint

Reply:
Sorry, missed this. Have updated the same.